build/gulp/plugins/gulp-component-menu-behaviors.ts

```diff
@@ -38,26 +44,25 @@ export default () => {
       const behaviorVariantName = file.basename
       const behaviorName = path.basename(dir)
 
-      let description
+      let descriptionText
```

Comment:
nit: these two could be declared closer to the place where those are initialized (`if` statement) - i.e. on line 52.

Actually, you could use more expressive code here, with no `let`:

```js
  const fileContent = ..
  const blockComments = ...

  const variation = {
    name: behaviorVariantName
  }
  
  if (...) {
     ...
    variation.description = getTextFromCommentToken(commentTokens, 'description')
    variation.specification = getTextFromCommentToken(commentTokens, 'specification')
    ...
  }